src/schema/users.ts

```diff
@@ -661,7 +709,6 @@ export const resolvers: IResolvers<any, Context> = {
       { id }: { id: string },
       ctx: Context,
     ): Promise<GQLUserStats | null> => {
-      const isSameUser = ctx.userId === id;
```

Comment:
We no longer want this eval?

Reply:
Nope, we gonna show these metrics publicly for now